### PR TITLE
Fix `wallet nep5 transfer` command

### PR DESCRIPTION
**Pull request by fyrchik**
Fix small bugs and one critical regarding merkle root construction.

### CODE REVIEW

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -20,13 +20,13 @@ import (
...
 var (
-	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -p) flags")
+	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -w) flags")
 	errPhraseMismatch = errors.New("the entered pass-phrases do not match. Maybe you have misspelled them")
 )
 
 var (
 	walletPathFlag = cli.StringFlag{
-		Name:  "path, p",
+		Name:  "path, w",
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:29]: `wallet, w`? And we have `-p` mentioned in `docs`, so we need to update them too.

File: cli/wallet/nep5.go [hunk 2]
```diff
@@ -185,7 +185,7 @@ func getMatchingTokenAux(get func(i int) *wallet.Token, n int, name string) (*wa
 	var count int
 	for i := 0; i < n; i++ {
 		t := get(i)
-		if t != nil && (t.Name == name || t.Symbol == name || t.Address() == name || t.Hash.StringLE() == name) {
+		if t != nil && (t.Name == name || t.Symbol == name || t.Address() == name || t.Hash.StringBE() == name) {
```
Comment (roman-khimov) [inline cli/wallet/nep5.go:188]: `name` comes from the user and user is always operating with LE hashes, so this change is wrong.

File: cli/wallet/nep5.go [hunk 3]
```diff
@@ -16,6 +17,13 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	neoTokenHash, _ = util.Uint160DecodeStringBE("3b7d3711c6f0ccf9b1dca903d1bfa1d896f1238c")
+	neoToken        = wallet.NewToken(neoTokenHash, "NEO", "neo", 0)
+	gasTokenHash, _ = util.Uint160DecodeStringBE("897720d8cd76f4f00abfa37c0edd889c208fde9b")
```
Comment (roman-khimov) [inline cli/wallet/nep5.go:23]: `rpc/client`, along with policy hashes.

File: cli/wallet/wallet.go (hunk 1 above)
Comment (fyrchik) [reply cli/wallet/wallet.go]: Fixed.

File: cli/wallet/nep5.go (hunk 3 above)
Comment (fyrchik) [reply cli/wallet/nep5.go]: Policy hash is already there.

Comment (fyrchik) [reply cli/wallet/nep5.go]: Fixed.

File: cli/wallet/wallet.go [hunk 4]
```diff
@@ -20,13 +20,13 @@ import (
...
 var (
-	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -p) flags")
+	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--wallet, -w) flags")
 	errPhraseMismatch = errors.New("the entered pass-phrases do not match. Maybe you have misspelled them")
 )
 
 var (
 	walletPathFlag = cli.StringFlag{
-		Name:  "path, p",
+		Name:  "wallet, w",
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:29]: So where is the documentation update for this?

Comment (fyrchik) [reply cli/wallet/wallet.go:29]: You mean `docs/cli.md`? Fixed. I've first grepped by full name.

Comment (roman-khimov) [reply cli/wallet/wallet.go:29]: Yep, that was missing.

**Pull request merged**